### ISSUE TITLE
CIT Resewn JSONLagPatch

### ISSUE THREAD

**Issue by wchristian** [Fabulously-Optimized/fabulously-optimized#759]
### CurseForge link

https://legacy.curseforge.com/minecraft/mc-mods/cit-resewn-jsonlagpatch

### CurseForge Mod Distribution

Allowed

### Modrinth link

https://modrinth.com/mod/cit-resewn-jsonlagpatch

### Source/other link

https://github.com/wchristian/CITResewnJSONLagPatch

### Mod file size

385.77

### License

(any other license)

### What it does

The same thing as CIT Resewn, but with some performance improvements.

### Why should it be in the modpack

Trying to play on Hypixel with Fabulously Optimized and the Hypixel+ resource pack is impossible without severe performance issues as things currently stand. This makes it possible.

### Why shouldn't it be in the modpack

The only downside over CIT Resewn is that it should be replaced with CIT Resewn again once the hiatus of the maintainer of that is over, and it has included performance improvements.

**Comment by Madis0** [comment]
I'm a bit skeptical for now, but maybe.

**Comment by ItsProfessional** [comment]
Considering that the project literally has the words "patch" in it, it should probably just be a PR instead of a seperate mod.

**Comment by Radplay** [comment]
> Considering that the project literally has the words "patch" in it, it should probably just be a PR to CIT resewn instead of a separate mod.

It is a PR: https://github.com/SHsuperCM/CITResewn/pull/320

**Comment by wchristian** [comment]
Yes, the whole point is that it has been PR'd and despite CIT Resewn in its current state having massive issues, which the patch strongly alleviates with very low impact, its author hasn't integrated it and in fact appears to be on a hiatus.

**Comment by Madis0** [comment]
I'd like to consider this for 1.20.4, if it gets updated for that version.

**Comment by Madis0** [comment]
#818 was added instead, as this one did not get updated in time.
